packages/payments-plugin/src/mollie/mollie-shop-schema.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from 'graphql-tag';
+
+export const shopSchema = gql`
+    type MolliePaymentIntentError {
```

Comment:
Implement the `ErrorResult` interface here for easier generic error handling on the client.

Reply:
Done, it now uses the errorCode `ORDER_PAYMENT_STATE_ERROR` when an order is not 'ready' for payment intent.